products/pixel.md

```diff
@@ -36,6 +44,7 @@ releases:
     eol: 2028-06-20
     discontinued: false
     link: https://en.wikipedia.org/wiki/Pixel_Tablet
+    supportedAndroidVersions: 13
```

Comment:
Same story here (and on more recent devices). Even if I don't know if Android 14 is available yet on this device I think we should put 13+ as Google promised 3 years of Android updates.

Reply:
I don't mind the 13+ for easier management, but is it consistent with every other pages where we write explicitly only the updates that are officially out ?

Reply:
You have a point @paulverbeke.

So let's drop the 13+, and check on https://www.gsmarena.com/ for the android support (where Pixel Tablet and 7a as "upgradable to Android 14" - did not check for other models). WDYT ?

Reply:
LGTM !